### ISSUE TITLE
Could not load INI file for version 2019.1.10f1 on linux

### ISSUE THREAD

**Issue by KonH** [DragonBox/u3d#366]
### Issue Checklist

- [x] I'm using the latest u3d version
- [x] I ran `u3d --help`
- [x] I read the [README](https://github.com/DragonBox/u3d/blob/master/README.md)
- [x] I made sure that a similar [issue](https://github.com/DragonBox/u3d/issues) doesn't exit - I found https://github.com/DragonBox/u3d/issues/242 but it looks like solved issue

### Issue Description

I've got an installation issue, which happens inside docker container:
```
Run program 'u3d' with command: 'install 2019.1.10f1 -p Unity,WebGL'
Stdout:
Could not load INI file for version 2019.1.10f1 on linux: Could not parse INI data (Could not parse line: "<HTML><HEAD>")
Stderr:
error: undefined method `keys' for nil:NilClass. Use --trace to view backtrace
Exit code is 1
```

This behavior is shown only when I run build in Travis environment. When I use the same container on a local machine with macOS, I can successfully install the same version of Unity.
A version of u3d from the master is used.
Maybe I didn't correctly understand https://github.com/DragonBox/u3d/issues/242 comments, but it looks like this problem should be fixed for now.

See full info here - https://travis-ci.org/KonH/UnityCiPipeline/builds/561162947
Related Dockerfile: https://github.com/KonH/UnityCiPipeline/blob/master/Docker/Dockerfile

Can I do something to clarify issue (--trace will be useful, for example)?

**Comment by KonH** [comment]
Anyone? I hope it can be resolved sometimes later.

**Comment by niezbop** [comment]
Hey @KonH, sorry to get back to you so late.

Did you find a solution to your issue? I'm not sure that this is u3d-related since I can't find the version `2019.1.10f1` for Linux anywhere on the net right now.

Is this issue still occuring with Unity 2019.1.14f1?

I seem to be getting the packages just fine, which would mean that the .ini are fine:
```
paulniezborala@Pauls-MacBook-Pro u3d (fix/windows/get_installed_versions_post_2019) $ u3d available -o linux -p --force --no-central -u 2019.1
Cache is out of date. Updating cache for Linux
Loading Unity releases
Loading Unity latest releases
Found 8 latest releases.
Found 92 releases.
Version 2019.1.0f2: http://beta.unity3d.com/download/292b93d75a2c/
Packages:
 - Unity
 - Android
 - iOS
 - Mac-Mono
 - WebGL
 - Windows-Mono
 - Facebook-Games
Version 2019.1.14f1: https://download.unity3d.com/download_unity/148b5891095a/
Packages:
 - Unity
 - Android
 - iOS
 - Mac-Mono
 - WebGL
 - Windows-Mono
 - Facebook-Games
```

I'll be closing this issue for now, feel free to reopen it if the issue persists with the newer versions of Unity.

If it does,
> Can I do something to clarify issue (--trace will be useful, for example)?
Yep! It would help a lot. You can specify  both `--trace` and `--verbose`

**Comment by KonH** [comment]
Thanks for your reply, it works fine now for Unity 2019.1.14f1.
